Source/WebKit/UIProcess/Automation/protocol/BidiBrowser.json

```diff
@@ -1,12 +1,29 @@
 {
     "domain": "BidiBrowser",
     "exposedAs": "browser",
-    "description": "The browser module contains commands for managing the remote end browser process.",
     "condition": "ENABLE(WEBDRIVER_BIDI)",
+    "description": "The browser module contains commands for managing the remote end browser process.",
+    "spec": "https://w3c.github.io/webdriver-bidi/#module-browser",
+    "tests": "https://github.com/web-platform-tests/wpt/tree/master/webdriver/tests/bidi/browser",
```

Comment:
NIT: `"tests"` makes me think it's a link to something else in WebKit source, so how about `"wpt"` or `"testURL"` or something to clarify that it's external?